api/app.ts

```diff
@@ -18,38 +25,26 @@ import {
   AUTH0_AUDIENCE,
   AUTH0_DOMAIN,
   NODE_ENV,
-  PORT,
   MONGOHQ_URL,
   SESSION_SECRET
 } from './utils/env';
 
-const envToLogger = {
-  development: {
-    transport: {
-      target: 'pino-pretty',
-      options: {
-        translateTime: 'HH:MM:ss Z',
-        ignore: 'pid,hostname'
-      }
-    },
-    level: 'debug'
-  },
-  // TODO: is this the right level for production or should we use 'error'?
-  production: { level: 'fatal' },
-  test: false
-};
-
-const fastify = Fastify({
-  logger: envToLogger[NODE_ENV]
-}).withTypeProvider<TypeBoxTypeProvider>();
-
-export type FastifyInstanceWithTypeProvider = typeof fastify;
+export type FastifyInstanceWithTypeProvider = FastifyInstance<
+  RawServerDefault,
+  RawRequestDefaultExpression,
+  RawReplyDefaultExpression,
+  FastifyBaseLogger,
+  TypeBoxTypeProvider
+>;
```

Comment:
Just so we are on the same page (out of interest): Why this?

Reply:
Limitations of my TypeScript skills, basically.  We want to correctly type the `fastify` instance, but we're using the `TypeBoxTypeProvider`, so we can't just use `FastifyInstance`'s defaults as they include `FastifyTypeProviderDefault`.

Before we could just use `typeof fastify`, but now `fastify` is created in a function and you can't export from functions.

Anyway, if there's a neater way to generate this, I don't know it.